contrib/jackc/pgx.v5/pgx_tracer.go

```diff
@@ -175,6 +183,41 @@ func (t *pgxTracer) TraceConnectEnd(ctx context.Context, data pgx.TraceConnectEn
 	finishSpan(ctx, data.Err)
 }
 
+func (t *pgxTracer) TraceAcquireStart(ctx context.Context, pool *pgxpool.Pool, _ pgxpool.TraceAcquireStartData) context.Context {
+	if !t.cfg.traceAcquire {
+		return ctx
+	}
+	opts := t.spanOptions(pool.Config().ConnConfig, operationTypeAcquire, "")
+	_, ctx = tracer.StartSpanFromContext(ctx, "pgx.acquire", opts...)
```

Comment:
```suggestion
	_, ctx = tracer.StartSpanFromContext(ctx, "pgx.pool.acquire", opts...)
```

Minor suggestion here, to better distinguish the `pgxpool` specific spans from the regular client ones.